pkg/pvresizing/pv.go

```diff
@@ -0,0 +1,27 @@
+package pvresizing
+
+const (
+	Name = "prometheus"
+
+	PrometheusStorageSizeSmall  PrometheusStorageSizeType = "small"
+	PrometheusStorageSizeMedium PrometheusStorageSizeType = "medium"
+	PrometheusStorageSizeLarge  PrometheusStorageSizeType = "large"
+)
+
+type PrometheusStorageSizeType string
+
+// PrometheusDiskSize returns the desired disk size based on the
+// value of annotation monitoring.giantswarm.io/prometheus-disk-size
+func PrometheusDiskSize(v string) string {
+	value := PrometheusStorageSizeType(v)
```

Comment:
Can you name v something more meaningful? Also, documenting what this parameter is makes sense :)